assets/src/components/admin/admin_tables.tsx

```diff
@@ -417,7 +418,7 @@ const ElevatorV2ScreensTable = (): JSX.Element => {
             newRow.app_params.accessible_path_image_here_coordinates.x = value;
             return newRow;
           },
-          Cell: EditableCell,
+          Cell: EditableNumberInput,
```

Comment:
Is this a related change?

Reply:
It's related in the sense that Knip reported `EditableNumberInput` as an unused export, and this caused me to realize there was actually a place we could be using it.